<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImportPoint</class>
 <widget class="QDialog" name="DialogImportPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Importer point</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="topMargin">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="fichierpointsLabel">
       <property name="text">
        <string>Fichier points</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QgsFileWidget" name="select_points_action"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_crs">
       <property name="text">
        <string>Système de coordonnées des points (voir sur le plan PDF)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QgsProjectionSelectionWidget" name="mQgsProjectionSelectionWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_import_points">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>button_import_points</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
